package/skylark_download_daemon/src/skylark_download_daemon.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2017 Swift Navigation Inc.
+ * Contact: Mark Fine <mark@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <libskylark.h>
+#include <sbp_zmq.h>
+
+//
+// Download Daemon - connects to Skylark and receives SBP messages.
+//
+
+// TODO (mookerji): Command line stuff?
```

Comment:
Hmm, thinking about this more - maybe the idea would be that the configuration passed on the command line overrides what's in the settings, but if there's no configuration then it reads from settings and gets update from settings or something like that. If it leads to complicated logic, probably not worth doing.